apps/codeforafrica/src/components/ErrorPage/ErrorPage.js

```diff
@@ -0,0 +1,35 @@
+import Box from "@mui/material/Box";
+
+import ErrorHero from "@/codeforafrica/components/ErrorHero";
+import Page from "@/codeforafrica/components/Page";
+import RelatedStories from "@/codeforafrica/components/RelatedStories";
+
+function ErrorPage({ sections, ...props }) {
+  return (
+    <Page {...props}>
+      <ErrorHero {...props} />
```

Comment:
Isn't `Hero` just another `section`? This is how am expecting us to handle it: https://github.com/CodeForAfrica/ui/blob/4ad402289bc87dc01050b5997780054c0f3e9eb5/apps/codeforafrica/src/pages/about/index.page.js#L30-L32

Let me know if there is a reason to pass the whole `props`.